.travis.yml

```diff
@@ -46,7 +46,7 @@ script:
   - tox
   - ./tests/system_test.sh
 after_success:
-  - if [[ "$COVERAGE" == "true" ]]; then py.test tests; coveralls || echo "failed"; fi
+  - if [[ "$COVERAGE" == "true" ]]; then tox; coveralls || echo "failed"; fi
```

Comment:
Sorry @FlorianWilhelm, I might have asked this in the past, but what would be the implications of removing this second round of executing the tests and just running `coveralls`?

This help to speed up the tests to run in the CI...

Reply:
Hm.. actually I don't know anymore, maybe it's a copy & paste mistake. Does it also just work by only calling `coveralls`? If this is the case we should remove it.

Reply:
Let's check :)

Reply:
Well, seems to be working: https://coveralls.io/builds/17322542